Paintroid/src/main/java/org/catrobat/paintroid/ui/LayerAdapter.kt

```diff
@@ -90,17 +90,23 @@ class LayerAdapter(val presenter: LayerContracts.Presenter) : BaseAdapter(), Lay
     internal class LayerViewHolder(private val itemView: View, private val layerPresenter: LayerContracts.Presenter) : LayerContracts.LayerViewHolder {
         private val layerBackground: LinearLayout = itemView.findViewById(R.id.pocketpaint_item_layer_background)
         private val imageView: ImageView = itemView.findViewById(R.id.pocketpaint_item_layer_image)
-        private var bitmap: Bitmap? = null
+        private var currentBitmap: Bitmap? = null
         private val checkBox: CheckBox = itemView.findViewById(R.id.pocketpaint_checkbox_layer)
 
         companion object {
             private const val RESIZE_LENGTH = 400f
         }
 
-        override fun setSelected(position: Int, bottomNavigationViewHolder: BottomNavigationViewHolder, defaultToolController: DefaultToolController) {
+        override val bitmap: Bitmap?
+            get() = currentBitmap
+
+        override val view: View
+            get() = itemView
+
+        override fun setSelected(position: Int, bottomNavigationViewHolder: BottomNavigationViewHolder?, defaultToolController: DefaultToolController?) {
```

Comment:
as discussed these parameters will be non nullable when refactoring the LayerAdapter